examples/meta_project/rust/sub1/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+# SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0
+[package]
+name = "sub1"
+version = "0.1.0"
+authors = [
+  "Andrew Hayzen <andrew.hayzen@kdab.com>",
+]
+edition = "2021"
+license = "MIT OR Apache-2.0"
+
+# This will instruct Cargo to create an rlib our main crate is the staticlib
+[lib]
+crate-type = ["rlib"]
```

Comment:
this fails with
```
error: link modifiers combination `+bundle,+whole-archive` is unstable when generating rlibs
```

Reply:
I tested with nightly and this error still occurs.

Reply:
You need `#![feature(packed_bundled_libs)]` to ungate it if you can use nightly.

Reply:
Thanks! With the nightly compiler and that feature enabled this works :tada: 

Now the question is whether this will ever become stable, whether there is a timeline for this, and if we can help in any way.